### PR TITLE
{bio}[foss/2019a] Seurat v3.1.2

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node375.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/d97e2f75f360bb630ba9cabb5a3fa06b for a full test report.

**Comment by verdurin** [comment]
@smoors fancy adding this to `R-3.6.2` and `foss-2019b` too ...?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/c961cbe99f3c0b684476450aa167cb5d for a full test report.

**Comment by verdurin** [comment]
Though I suppose it depends on `Bioconductor` which hasn't yet been updated. Hmm.

File: easybuild/easyconfigs/s/Seurat/Seurat-3.1.2-foss-2019a-R-3.6.0.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }),
+    ('uwot', '0.1.5', {
+        'checksums': ['3253b44c5c7d9ecff25511ad1632a8e924677f9e9a1a8453088de09e90e2e95f'],
+    }),
+    (name, version, {
+        'checksums': ['446a5baf63efdb30ef86af011f94e989597d72787f1d70c39eb32eabd3883854'],
+    }),
+]
+
+modextrapaths = {'R_LIBS': ''}
```
Comment (verdurin) [inline easybuild/easyconfigs/s/Seurat/Seurat-3.1.2-foss-2019a-R-3.6.0.eb]: @smoors why did you add this? Is it because it's a bundle now instead of a package?

Comment (smoors) [reply easybuild/easyconfigs/s/Seurat/Seurat-3.1.2-foss-2019a-R-3.6.0.eb]: @verdurin indeed. R_LIBS is automatically set for RPackage, but not for Bundle.
I guess we could create an RBundle to fix this.

**Comment by smoors** [comment]
@verdurin sure, I can add it once #9639 has been merged.

**Comment by verdurin** [comment]
Going in, thanks @smoors!
